sched/wdog/wd_start.c

```diff
@@ -74,6 +74,12 @@
 #  define CALL_FUNC(func, arg) func(arg)
 #endif
 
+/****************************************************************************
+ * Private Data
+ ****************************************************************************/
+
+static bool g_wdexpiration_context;
```

Comment:
Change to g_in_wd_expiration?

Reply:
I will change to volatile, but since the wd_start is included indirectly via wd expiration callback the flag value will be checked anyway

Reply:
Fixed